docker-compose.yaml

```
@@ -18,7 +18,7 @@ services:
     env_file:
       - .env
     environment:
-      - MONGO_INITDB_ROOT_USERNAME=${MONGODB_INITDB_ROOT_USERNAME}
+      - MONGO_INITDB_ROOT_EMAIL=${MONGODB_INITDB_ROOT_EMAIL}
```

As above `MONGO_INITDB_ROOT_USERNAME` refers to the root level access to the database instance. This variable is named explicitly and CANNOT be changed because it is a keyword that Self-Managed Deployments of MongoDB use to give instance access. The purpose of the docker-compose.yaml file is to create a Docker compose project with the correct environment variables, and changing them will cause the environment to be not established correctly. This particular line uses the `.env` file to populate this in the container environment.

---

Please revert this change
```suggestion
      - MONGO_INITDB_ROOT_USERNAME=${MONGODB_INITDB_ROOT_USERNAME}
```